MIGRATION.md

```diff
@@ -115,7 +115,7 @@ addParameters({
     theme: create({
       brandTitle: 'Storybook',
       brandUrl: 'https://storybook.js.org'
-    }, 'normal'),
+    }, 'light'),
```

Comment:
The second argument to `create` is an object: https://github.com/storybooks/storybook/blob/next/lib/theming/src/create.ts#L113